bundle-size/test/webhooks.test.js

```diff
@@ -221,6 +221,11 @@ describe('bundle-size webhooks', () => {
         name: 'pull_request',
         payload: pullRequestPayload,
       });
+
+      // Silence expected error messages during testing.
+      const logLevel = probot.logger.level();
```

Comment:
^

Reply:
In this case, since you want the error to propagate outside the Probot handler, the message is automatically logged by Probot's error event handler (as opposed to the app/context/request loggers). As best as I can tell, there's no direct access to these methods (they're either private or non-explicit, somehow embedded in Probot error catching code). The change in log level still takes effect, though.

Reply:
Ack